tests/cases/compiler/tslibMultipleMissingHelper.ts

```diff
@@ -0,0 +1,44 @@
+// @filename: /tsconfig.json
+{
+    "compilerOptions": {
+        "strict": true,
+        "target": "ES2016",
+        "importHelpers": true,
+        "module": "commonjs",
+    }
+}
+
+// @filename: /package1/index.ts
+export {};
+async function foo(): Promise<void> {}
+async function bar(): Promise<void> {}

```

Comment:
Maybe put another file that needs a helper inside `package1` so we can see (if I understand correctly) that the error doesn’t duplicate across _files_ that resolve to the same copy of `tslib`?